catroid/src/main/java/org/catrobat/catroid/ui/SettingsActivity.java

```diff
@@ -368,6 +385,14 @@ public static void setAutoCrashReportingEnabled(Context context, boolean value)
 		SharedPreferences.Editor editor = getSharedPreferences(context).edit();
 		editor.putBoolean(SETTINGS_CRASH_REPORTS, value);
 		editor.commit();
+
+		if (!value) {
+			CrashReporter.disableCrashlytics();
+			Log.e(TAG, "Crashlytics Disabled");
+		} else {
+			CrashReporter.initialize(context);
+			Log.e(TAG, "Crashlytics Enabled");
```

Comment:
Please don't log it as `e`.
'Crashlytics Enabled' may be misleading. Crashlytics may be enabled depending on the flags in the CrashReporter. So it would be better to move the logging up one line in this block and rename it to `Enabling Crashlytics`.

However, the CrashReporter already does some logging. I'm not sure if it's necessary to log twice…